src/Microsoft.AspNetCore.Server.Kestrel/Internal/Networking/UvStreamHandle.cs

```diff
@@ -118,17 +118,17 @@ public void ReadStop()
             _uv.read_stop(this);
         }
 
-        public int TryWrite(Libuv.uv_buf_t buf)
+        public void TryWrite(Libuv.uv_buf_t buf)
         {
-            return _uv.try_write(this, new[] { buf }, 1);
+            _uv.try_write(this, new[] { buf }, 1);
```

Comment:
TryWrite returns the number of bytes written to a socket or an error.

I don't think that removing the returned value is really something one wants to do with this API. It is currently not used, but in the future, if one were to use it, one would check how much of the buffer has been written by using the returned value and eventually queue the rest of the buffer for writing with a normal Write call.